docs/src/api/class-page.md

```diff
@@ -2683,6 +2683,11 @@ An object which specifies clipping of the resulting image. Should have the follo
 Hides default white background and allows capturing screenshots with transparency. Not applicable to `jpeg` images.
 Defaults to `false`.
 
+### option: Page.screenshot.blackout
+- `blackout` <[Array]<[Locator]>>
```

Comment:
s/blackout/hide|redact|cover|overlay|...

Reply:
We discussed this last week and so far settled upon `mask`.